akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/scaladsl/ActorTestKit.scala

```diff
@@ -135,6 +136,9 @@ final class ActorTestKit private[akka] (val name: String, val config: Config, se
   implicit def testKitSettings: TestKitSettings =
     settings.getOrElse(TestKitSettings(system))
 
+  // avoid slf4j noise by touching it first from single thread #28673
+  LoggerFactory.getLogger(getClass).debug("Starting ActorTestKit")
```

Comment:
include `name` for additional value

Reply:
Had bad luck thinking when I passed the class... fixed.